package/kedro_viz/models/experiment_tracking.py

```diff
@@ -78,7 +77,7 @@ class TrackingDatasetModel:
     dataset_name: str
     # dataset is the actual dataset instance, whereas dataset_type is a string.
     # e.g. "tracking.metrics_dataset.MetricsDataSet"
-    dataset: AbstractVersionedDataset
+    dataset: "AbstractVersionedDataset"
```

Comment:
I'm not 100 % sure (please check) but this might need `from __future__ import annotations`

Reply:
It's good from 3.7. I think with `from __future__ import annotations`, it's forward reference by default IIRC?